src/App.css

```diff
@@ -1,4 +1,116 @@
 body {
```

Comment:
General mistake: you didn't use styles from figma. Result should be matched design. All styles of any elements you can see if click on an element.

Reply:
<img width="702" alt="Screenshot 2024-05-11 at 11 40 48" src="https://github.com/PixelsPioneer/fe-react-2024/assets/102523692/bece2e76-3356-46a1-b03e-8ed54eaa2e75">
To utilize the space more effectively, widen the text on the page.